app/src/main/java/swati4star/createpdf/util/ImageUtils.java

```diff
@@ -221,18 +220,14 @@ public Bitmap toGrayscale(Bitmap bmpOriginal) {
      * @param filename    - name of the file
      * @param finalBitmap - bitmap to save
      */
-    public static String saveImage(String filename, Bitmap finalBitmap) {
+    public static String saveImage(String filename, Bitmap finalBitmap, String myDir) {
```

Comment:
update the comment above it also to include the description of new parameter you added to this function

Reply:
Resolved in commit 306fbab